### DISCUSSION TITLE
Remove EOF character displayed

### DISCUSSION THREAD

**Discussion by AlexanderBruland** [JetBrains/ideavim#784] [Q&A] [answered]
Hello! I have the most anoying setting in ideavim that I can't figure out how to turn off. It goes away when I go to tools and turn off ideavim, but comes back automaticly from boot. I have turned of whitespaces and everything in the webstorm settings. I have tried googling, and 10 conversations with chatgpt but I still can't figure it out. 

Picture: 
![image](https://github.com/JetBrains/ideavim/assets/110822310/681d3dd5-7e6e-4739-b37e-942769f0f8fa)

.ideavimrc::

set number relativenumber
set visualbell
set hlsearch incsearch ignorecase

let mapleader=","

map \<leader\>vm :vsplit C:\Users\Alexander\.ideavimrc \<CR\>

nnoremap \<leader\>w :w! \<CR\>
nnoremap \<leader\>q :q \<CR\>

set cb

set clipboard+=unnamed
set matchpairs

map \<leader\>q :q \<CR\>

set NERDTree

**Comment by AlexanderBruland** [comment, marked as answer]
Update: Removed "set hlsearch incsearch ignorecase" from .ideavimrc and now it does not show EOL character. No idea why. 

Keywords for search because I couldnt find anything: End Of Line, character, $, EOF, IdeaVim, Webstorm.

**Reply by citizenmatt** [reply]
This is IdeaVim highlighting the last search term, which must have matched the end of the line, or CR or some such. The easy way to fix this is to search for something else. You can also use the `:nohlsearch` command to temporarily hide the highlight until the next search. I have this mapped to `<C-L>`.
